setup.cfg

```diff
@@ -1,6 +1,6 @@
 # This section is for flake8.
 [flake8]
-ignore = E203,W503
+ignore = E203,W503,H3
```

Comment:
A nit. How about leaving a comment that `H3` covers `hacking`s import statement guidelines and that they're ignored because of `isort` compatibility?

Reply:
@hvy Done!